epytope/Core/Result.py

```diff
@@ -56,46 +61,61 @@ class EpitopePredictionResult(AResult):
 
         EpitopePredictionResult
 
-        +--------------+-------------+-------------+-------------+-------------+
-        | Peptide Obj  | Method Name | Allele1 Obj | Allele2 Obj | Allele3 Obj |
-        +==============+=============+=============+=============+=============+
-        | Peptide1     | Method 1    |    0.324    |     0.56    |    0.013    |
-        +              +-------------+-------------+-------------+-------------+
-        |              | Method 2    |     20      |      15     |     23      |
-        +--------------+-------------+-------------+-------------+-------------+
-        | Peptide2     | Method 1    |    0.50     |     0.36    |    0.98     |
-        +              +-------------+-------------+-------------+-------------+
-        |              | Method 2    |     26      |      10     |     50      |
-        +--------------+-------------+-------------+-------------+-------------+
+        +----------------+-------------------------------+-------------------------------+
+        |  Allele        |          Allele Obj 1         |          Allele Obj 2         | 
+        +- - - - - - - - +- - - - - - - -+- - - - - - - -+- - - - - - - -+- - - - - - - -+
+        |  Method        |    Method 1   |    Method 2   |    Method 1   |    Method 2   |
+        +- - - - - - - - +- - - -+- - - -+- - - -+- - - -+- - - -+- - - -+- - - -+- - - -+
+        | ScoreType      | Score |  Rank | Score |  Rank | Score |  Rank | Score |  Rank |
+        +- - - - - - - - +- - - -+- - - -+- - - -+- - - -+- - - -+- - - -+- - - -+- - - -+
+        |  Peptides      |       |       |       |       |       |       |       |       |
+        +================+=======+=======+=======+=======+=======+=======+=======+=======+
+        | Peptide Obj 1  |  0.03 |  57.4 |  0.05 |  51.1 |  0.08 |  49.4 |  0.73 |  3.12 |
+        +----------------+-------+-------+-------+-------+-------+-------+-------+-------+
+        | Peptide Obj 2  |  0.32 |  13.2 |  0.31 |  14.1 |  0.25 |  22.1 |  0.11 |  69.1 |
+        +----------------+-------+-------+-------+-------+-------+-------+-------+-------+
 
     """
 
-    def filter_result(self, expressions):
+    def filter_result(self, expressions, scoretype=False):
         """
         Filters a result data frame based on a specified expression consisting of a list of triple with
-        (method_name, comparator, threshold). The expression is applied to each row. If any of the columns fulfill
-        the criteria the row remains.
+        (method_name/scoretype_name, comparator, threshold) and a boolean if the scoretype is specified.
```

Comment:
I was just wondering if it would be cleaner to have the first part of the triple only for `method_name` and the additional parameter `scoretype` with a default value `Score`.

Reply:
I changed this method according to your suggestion. Please have a look again :)

Reply:
This looks still the same to me, could you please doublecheck?